.travis/upload-nightly-wheels.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# This script gets newest wheels from S3 and uploads them PyPI
+
+WHEEL_DIR="./.whl"
+mkdir -p $WHEEL_DIR
+
+# Clone Ray to get commit hash
+git clone https://github.com/ray-project/ray.git
+
+# Sync newest wheels from AWS
+COUNT=0
+while ! [[ $WHEELS ]]; do
```

Comment:
I think we want to do this until all **10** wheels are present (e.g., what if the Linux wheels failed to build).

Reply:
+1